Items/ladder.json

```diff
@@ -3,7 +3,7 @@
     "description": "It's a small aluminum ladder. Useful for reaching high places.",
     "secondaryDescription": "You drop the ladder underneath the attic hatch. You open the hatch and climb up.",
     "itemInteractions": {
-        "Ladder": "None",
+        "ladder": "None",
         "Flashlight": "None",
```

Comment:
The rest of ladder.json needs to be updated.

Reply:
Whoops, got distracted lol. Done.